tests/WorkflowJob.py

```diff
@@ -0,0 +1,80 @@
+############################ Copyrights and license ############################
+#                                                                              #
+# Copyright 2021 Jeppe Fihl-Pearson <jeppe@tenzer.dk>                          #
+#                                                                              #
+# This file is part of PyGithub.                                               #
+# http://pygithub.readthedocs.io/                                              #
+#                                                                              #
+# PyGithub is free software: you can redistribute it and/or modify it under    #
+# the terms of the GNU Lesser General Public License as published by the Free  #
+# Software Foundation, either version 3 of the License, or (at your option)    #
+# any later version.                                                           #
+#                                                                              #
+# PyGithub is distributed in the hope that it will be useful, but WITHOUT ANY  #
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS    #
+# FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more #
+# details.                                                                     #
+#                                                                              #
+# You should have received a copy of the GNU Lesser General Public License     #
+# along with PyGithub. If not, see <http://www.gnu.org/licenses/>.             #
+#                                                                              #
+################################################################################
+
+import datetime
+
+from . import Framework
+
+
+class WorkflowJob(Framework.TestCase):
+    def setUp(self):
+        super().setUp()
+        self.repo = self.g.get_repo("PyGithub/PyGithub")
+        self.job = self.repo.get_workflow_run(4205440316).jobs()[0]
```

Comment:
Let's assert the number of jobs as well:
```suggestion
        jobs = self.repo.get_workflow_run(4205440316).jobs()
        self.assertEqual(len(jobs), 5)
        self.job = jobs[0]
```